Manuals updated and translated

JoelvdWeg opened the pull request.

JoelvdWeg committed:
Manuals updated and translated

n-vr committed:
feat: create language redirect

n-vr commented:
I added a language redirect that routes /manual/twomes-p1-reader-firmware to /manual/twomes-p1-reader-firmware/<language_code>

JoelvdWeg committed:
shared images

both languages use the same image folder now, saves some space

JoelvdWeg committed:
Merge branch 'p1-reader-manuals' of https://github.com/energietransitie/energietransitie.github.io into p1-reader-manuals

n-vr committed:
docs: bring up-to-date with twomes-p1-reader repo

Newer documentation was available in the twomes-p1-reader repo.
This is now copied here manually.
